src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ChunkOutputWidget.ui.xml

```diff
@@ -26,6 +26,17 @@
       right: 35px;
       transition: transform 400ms ease, -webkit-transform 400ms ease;
    }
+
+   .popout 
+   {
+      right: 58px;
+      width: 14px;
+      height: 14px;
+      top: 0px;
+      filter: grayscale(100%);
```

Comment:
I don't think `filter` is supported in IE (and may not work in QtWebKit either). It may be better to make a grayscale version of the PNG.